types/three/src/renderers/webxr/WebXR.d.ts

```diff
@@ -1,56 +1,111 @@
+/**
+ * Available session modes
+ */
 export type XRSessionMode = 'inline' | 'immersive-vr' | 'immersive-ar';
 
-export type XRReferenceSpaceType = 'viewer' | 'local' | 'local-floor' | 'bounded-floor' | 'unbounded';
+/**
+ * Reference space types
+ */
+export type XRReferenceSpaceType = 'viewer' | 'local' | 'local-floor' | 'unbounded';
+export type XRBoundedReferenceSpaceType = 'bounded-floor';
```

Comment:
calling `requestReferenceSpace` with `bounded-floor` returns `XRBoundedReferenceSpace` instead of `XRReferenceSpace` .
Strictly, this type does not exist in WebXR but the naming and typing feels reasonable enough.

See: https://developer.mozilla.org/en/docs/Web/API/XRSession/requestReferenceSpace

Reply:
See the implementation, this is not true.